polars/polars-core/src/fmt.rs

```diff
@@ -461,24 +461,29 @@ impl Display for DataFrame {
                 };
                 (s, lower_bounds)
             };
-            let tbl_lower_bounds =
+
+            let col_width_exact =
+                |l: usize| ColumnConstraint::Absolute(comfy_table::Width::Fixed(l as u16));
+            let col_width_lower_bound =
                 |l: usize| ColumnConstraint::LowerBoundary(comfy_table::Width::Fixed(l as u16));
+            // let col_width_upper_bound =
```

Comment:
Commented out code. Don't we use this anymore?

Reply:
Ahh, minor oversight there - it’s new; I’m going to use it in a follow-up patch (we’ve not previously set upper bounds).

Reply:
Let's get rid of the commented out code for now, keep the main branch clean. Then I'll approve this!

Reply:
Done; had to get off the side of a Swiss mountain first :))